city_metrix/layers/building_classifier.py

```diff
@@ -0,0 +1,181 @@
+import xarray as xr
+import ee
+import numpy as np
+from rasterio.enums import Resampling
+from geocube.api.core import make_geocube
+import pandas as pd
+import geopandas as gpd
+from sklearn.tree import DecisionTreeClassifier, plot_tree
+from sklearn.metrics import accuracy_score
+import matplotlib.pyplot as plt
+
+from .layer import Layer, get_utm_zone_epsg
+from .esa_world_cover import EsaWorldCover, EsaWorldCoverClass
+from .urban_land_use import UrbanLandUse
+
+
+class BuildingClassifier(Layer):
+    def __init__(self, geo_file=None, **kwargs):
+        super().__init__(**kwargs)
+        self.geo_file = geo_file
+
+    def get_data(self):
+        buildings_sample = gpd.read_file(self.geo_file)
```

Comment:
what is this file?

Reply:
A training dataset of 200 buildings https://github.com/wri/cities-heat/blob/main/data/buildings-sample-classed_LA.geojson